kubernetes/helm/collabora-online/values.yaml

```diff
@@ -18,25 +18,26 @@ global:
       port: 9980
       path: "/lool/getMetrics"
     service:
-      name: libreoffice-online
-      type: ClusterIP
+      name: collabora-online
+      type: NodePort # Use ClusterIP instead of NodePort if you are using standalone cluster structure 
       externalPort: 9980
       internalPort: 9980
   ingress: {}
 
-replicaCount: 3
+replicaCount: 3 # This is the amount of how many instances of loolwsd will run
+
 image:
   pullPolicy: Always
-  repository: libreoffice/online
-  tag: master
+  repository: collabora/code
+  tag: latest
 
 probes:
   urlScheme: HTTP # either HTTP or HTTPS
   port: 9980
   readiness:
     activate: true
     path: /
-    initialDelaySeconds: 180
+    initialDelaySeconds: 45
```

Comment:
why 180 -> 45 ?

Reply:
well I thought 3 minutes would be too much to wait just for loolwsd service to initialize

Reply:
and in fact it was while testing